tools/src/main/java/org/apache/kafka/tools/GroupsCommand.java

```diff
@@ -41,6 +39,10 @@
 import java.util.concurrent.TimeUnit;
 import java.util.stream.Stream;
 
+import joptsimple.ArgumentAcceptingOptionSpec;
+import joptsimple.OptionSpec;
+import joptsimple.OptionSpecBuilder;
+
```

Comment:
Side cleanup from running `spotlessApply`

Reply:
I guess I should make it a habit to run that

Reply:
Me too